growth/too/tasks/scheduler.py

```diff
@@ -225,3 +232,81 @@ def get_decam_dict(data_row, queue_name, cnt, nrows,
     decam_dict["wait"] = "False"
 
     return decam_dict
+
+
+def get_growthindia_table(json_data, sunrise_horizon=-12, horizon=20,
+                          priority=10000, domesleep=100):
+
+    t = Table(rows=json_data['targets'])
+    coords = SkyCoord(ra=t['ra'], dec=t['dec'], unit=(u.degree, u.degree))
+    hanle = EarthLocation(lat=32.77889*u.degree,
+                          lon=78.96472*u.degree,
+                          height=4500*u.m)
+    iao = Observer(location=hanle, name="GIT", timezone="Asia/Kolkata")
+
+    twilight_prime = iao.sun_rise_time(Time.now(),
+                                       which="next",
+                                       horizon=sunrise_horizon*u.degree)\
+        - 12*u.hour
+    targets_rise_time = iao.target_rise_time(twilight_prime,
+                                             coords,
+                                             which="nearest",
+                                             horizon=horizon*u.degree)
+    targets_set_time = iao.target_set_time(targets_rise_time, coords,
+                                           which="next",
+                                           horizon=horizon*u.degree)
+    rise_time_IST = (targets_rise_time + 5.5*u.hour).isot
+    set_time_IST = (targets_set_time + 5.5*u.hour).isot
+    tend = targets_set_time
+    mooncoords = get_moon(tend, hanle)
+    sep = mooncoords.separation(coords)
+
+    t.add_column(Column(name='x', length=len(t), dtype='S10'))
```

Comment:
This can be done more concisely by using one of the many forms of the `Table` constructor.